README.md-tpl

```diff
@@ -82,59 +83,337 @@ Generate an API key by signing in to
+    UUID notificationId;
+    Optional<String> reference;
+    UUID templateId;
+    int templateVersion;
+    String templateUri;
+    String body;
+    Optional<String> fromNumber;
+
 ```
 
-Find `templateId` by clicking **API info** for the template you want to send.
+Otherwise the client will raise a `NotificationClientException`:
+
+<table>
+<thead>
+<tr>
+<th>message</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<pre>
+Status code: 400 {
+"errors":
+[{
+    "error": "TooManyRequestsError",
+    "message": "Exceeded send limits (50) for today"
+}]
+}
+</pre>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+Status code 400: {
+"errors":
+[{
+    "error": "BadRequestError",
+    "message": "Can"t send to this recipient using a team-only API key"
+]}
+}
+</pre>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+Status code: 400 {
+"errors":
+[{
+    "error": "BadRequestError",
+    "message": "Can"t send to this recipient when service is in trial mode
+                - see https://www.notifications.service.gov.uk/trial-mode"
+}]
+}
+</pre>
+</td>
+</tr>
+<tr>
+</tbody>
+</table>
+</details>
 
-If a template has placeholders, you need to provide their values in `personalisation`, for example:
+
+### Email:
 
 ```java
 HashMap<String, String> personalisation = new HashMap<>();
 personalisation.put("name", "Jo");
 personalisation.put("reference_number", "13566");
-NotificationResponse response = client.sendEmail(emailTemplateId,
-        emailAddress, personalisation);
+SendEmailResponse response = client.sendEmail(templateId, mobileNumber, personalisation, "yourReferenceString");
+```
+
+
+<details>
+<summary>
+SendEmailResponse
+</summary>
+
+If the request is successful, the SendEmailResponse is returned from the client. Attributes of the SendEmailResponse are listed below.
+
+```java
+	UUID notificationId;
+	Optional<String> reference;
+	UUID templateId;
+	int templateVersion;
+	String templateUri;
+	String body;
+	String subject;
+	Optional<String> fromEmail;
+
 ```
 
+Otherwise the client will raise a `NotificationClientException`:
+
+<table>
+<thead>
+<tr>
+<th>message</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<pre>
+Status code: 400 {
+"errors":
+[{
+    "error": "TooManyRequestsError",
+    "message": "Exceeded send limits (50) for today"
+}]
+}
+</pre>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+Status code 400: {
+"errors":
+[{
+    "error": "BadRequestError",
+    "message": "Can"t send to this recipient using a team-only API key"
+]}
+}
+</pre>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+Status code: 400 {
+"errors":
+[{
+    "error": "BadRequestError",
+    "message": "Can"t send to this recipient when service is in trial mode
+                - see https://www.notifications.service.gov.uk/trial-mode"
+}]
+}
+</pre>
+</td>
+</tr>
+<tr>
+</tbody>
+</table>
+</details>
+
+### Arguments
+#### `phoneNumber`
+The mobile number the SMS notification is sent to.
+
+#### `emailAddress`
+The email address the email notification is sent to.
```

Comment:
should this arguments section be split up into `sendSms` and `sendEmail`? so we can separate when you pass in `phoneNumber` vs `emailAddress`